src/components/projectSelect/index.tsx

```diff
@@ -83,3 +94,24 @@ export const ProjectSelect: React.FC<ProjectSelectProps> = ({
     />
   );
 };
+
+export const getProjects = (isProjectSettingsPage) => {
```

Comment:
Since this is a hook it should have the following naming convention `useHookName`  https://reactjs.org/docs/hooks-custom.html